### PR TITLE
chore: add ivy to ecosystem

### PR DESCRIPTION
**Pull request by mcnuttandrew**

### CODE REVIEW

**Comment by mcnuttandrew** [comment]
Hmm i'm not sure how to address the semantic pull request fail? Any help would be appreciated

File: site/ecosystem.md [hunk 1]
```diff
@@ -84,6 +85,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 - [Voyager clone](https://matyunya-simple-voyager-clone.ellx.app), a basic Voyager clone with step-by-step tutorial made with [Ellx](https://ellx.io).
 - [NL4DV](https://nl4dv.github.io/nl4dv/), a Python toolkit that generates analytic specifications (attributes + tasks + Vega-Lite visualizations) from natural language (NL) queries, helping people prototype NL systems for data visualization.
 
+
```
Comment (domoritz) [inline site/ecosystem.md:88]:
```suggestion
```

File: site/ecosystem.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 - [ColorBrewer-Lite](https://github.com/vis-au/colorbrewer), a fork of the ColorBrewer project that allows importing Vega-Lite specifications into the ColorBrewer interface to pick effective color schemes "in situ" for any color encoding.
 - [Emacs Vega View](https://github.com/applied-science/emacs-vega-view), a tool that allows one to view Vega visualizations directly within emacs, currently supporting specs written in JSON, elisp or clojure.
 - [Codimd](https://github.com/hackmdio/codimd), realtime collaborative markdown notes editor with support of various diagram syntaxes including [Vega-Lite](https://hackmd.io/c/codimd-documentation/%2F%40codimd%2Fextra-supported-syntax#Vega-Lite).
+- [Ivy](http://ivy-vis.netlify.app/), an Integrated Visualization Editing environment that wraps Vega-Lite (and other JSON-based declarative visualization grammars) as templates to facilitate reuse, exploration, and opportunistic creation. (Includes an in-app reproduction of [Polestar](https://github.com/vega/polestar))
```
Comment (domoritz) [inline site/ecosystem.md:26]:
```suggestion
- [Ivy](http://ivy-vis.netlify.app/), an Integrated Visualization Editing environment that wraps Vega-Lite (among other declarative visualization grammars) as templates to facilitate reuse, exploration, and opportunistic creation. Includes an in-app reproduction of [Polestar](https://github.com/vega/polestar) 
```

Comment (mcnuttandrew) [reply site/ecosystem.md]: nice

File: site/ecosystem.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 - [ColorBrewer-Lite](https://github.com/vis-au/colorbrewer), a fork of the ColorBrewer project that allows importing Vega-Lite specifications into the ColorBrewer interface to pick effective color schemes "in situ" for any color encoding.
 - [Emacs Vega View](https://github.com/applied-science/emacs-vega-view), a tool that allows one to view Vega visualizations directly within emacs, currently supporting specs written in JSON, elisp or clojure.
 - [Codimd](https://github.com/hackmdio/codimd), realtime collaborative markdown notes editor with support of various diagram syntaxes including [Vega-Lite](https://hackmd.io/c/codimd-documentation/%2F%40codimd%2Fextra-supported-syntax#Vega-Lite).
+- [Ivy](http://ivy-vis.netlify.app/), an Integrated Visualization Editing environment that wraps Vega-Lite (among other declarative visualization grammars) as templates to facilitate reuse, exploration, and opportunistic creation. Includes an in-app reproduction of [Polestar](https://github.com/vega/polestar)
```
Comment (domoritz) [inline site/ecosystem.md:26]:
```suggestion
- [Ivy](http://ivy-vis.netlify.app/), an Integrated Visualization Editing environment that wraps Vega-Lite (among other declarative visualization grammars) as templates to facilitate reuse, exploration, and opportunistic creation. Includes an in-app reproduction of [Polestar](https://github.com/vega/polestar).
```

**Pull request merged**